README.md

```diff
@@ -70,14 +70,28 @@ To activate Zen you then just have to add the following argument to your Java co
 ```
 Replace `/opt/zen` with your directory of choice.
 
-To use user-blocking and/or rate-limiting features, you will have to include the following Jarfile into your repository, for gradle :
+To use user-blocking and/or rate-limiting features, you will have to include the following Jarfile into your repository
+### Gradle
+Add the following code to your `build.gradle` file.
 ```gradle
 dependencies {
     # ...
     implementation files('/opt/zen/agent_api.jar')
     # ...
 }
 ```
+### Maven
+Add the following code to your `pom.xml` file.
+```xml
+<dependency>
+    <groupId>dev.aikido</groupId>
+    <artifactId>agent_api</artifactId>
+    <version>1.0</version>
```

Comment:
version

Reply:
version number should not matter when importing from jar, but maven requires it anyways, so 1.0.

Reply:
Wait according to maven docs it's optional

Reply:
https://maven.apache.org/plugins/maven-install-plugin/install-file-mojo.html